src/lib/Flasher.js

```diff
@@ -432,16 +440,20 @@ class Flasher {
       return null;
     }
 
-    if (this._missedChunks.size) {
```

Comment:
@haeferer - I think this is solved if we just remove this if-statement. We should always wait a bit for the missing chunk whether we have any or we don't.